app/src/main/java/com/example/memeitupapp/ui/gridmemes/GridMemesFragment.kt

```diff
@@ -23,20 +26,44 @@ class GridMemesFragment : Fragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        gridMemesViewModel = ViewModelProvider(this).get(GridMemesViewModel::class.java)
+        gridMemesViewModel.getMemesFromApi()
```

Comment:
rename the method. this is not a get because it returns nothing. it set liveData state

Reply:
renamed the method, along with other methods with non-descriptive names